github_copilot_seats/triggers.py

```diff
@@ -0,0 +1,28 @@
+from autokitteh.slack import slack_client
+import seats
+
+s = slack_client("slack_conn")
```

Comment:
Order the imports, and split into 3 blocks: standard library (none in this specific file), 3rd-party (including AK SDK), and project-internal.

```suggestion
from autokitteh.slack import slack_client

import seats

s = slack_client("slack_conn")
```